Source/JavaScriptCore/inspector/protocol/Console.json

```diff
@@ -34,6 +34,12 @@
             "enum": ["off", "basic", "verbose"],
             "description": "Level of logging."
         },
+        {
+            "id": "ClearReason",
+            "type": "string",
+            "enum": ["console-clear-api", "main-frame-navigation"],
```

Comment:
we should just have this be `"console"` or `"console-api"` to match other uses of things like this elsewhere

NIT: i wonder if just `"navigation"` would be enough 🤔

Reply:
I don't think `navigation` is enough here. We only send this for main frame navigations, and that feels like an important distinction.